public/js/main.js

```diff
@@ -141,11 +142,11 @@ listData.addEventListener("click", (event) => {
 });
 
 btnToggleAll.addEventListener("click", () => {
-  let count = todoList.reduce((count, item) => {
+  let countCompletedTasks = todoList.reduce((count, item) => {
```

Comment:
The variable should be a noun. Rename to `completedTasksCounter`

- [ ] rename